RELEASES.md

```diff
@@ -7,6 +7,14 @@ Releases are proposed and adopted or rejected using the usual [project governanc
 An anti-pattern that we want to avoid is heavy development or discussions "late cycle" around major releases.
 We want to build a community that is involved and communicates consistently through all releases instead of relying on "silent periods" as a judge of stability.
 
+## Release Process
```

Comment:
i also think runtime-spec specific process doesnt need to be in the same file as the generic oci release.md. Maybe these can just be in README ? Separating makes it easier to apply upstream Release.md changes too if necessary.